Reference/Routing/surface-controllers.md

```diff
@@ -1,34 +1,33 @@
 ---
 versionFrom: 7.0.0
-needsV8Update: "true"
```

Comment:
All concepts and code still apply to Umbraco 8, so this shouldn't require additional updates.